backend/src/events/board-events.ts

```diff
@@ -40,6 +40,17 @@ export function updateBoard(io: Server, client: Socket, roomId: string): void {
   });
 }
 
+function overwriteColumnIsBlurredState(board: RetroBoard): void {
```

Comment:
Could you explain what this function is doing specifically? Especially the `isBlurredStateMatches` part.

Reply:
This function handles the case, when a user blurred all non empty columns manually and then blurres the complete board.

Reply:
'isBlurredStateMatches' checks if all non empty columns are either all set to true or false. If the state of all columns matches, we apply it to the board state (the case when the board state has been changed by toggling all columns manually).

Reply:
I refactored the method to ensure better readability